projects/ngx-extended-pdf-viewer/src/lib/pinch-on-mobile-support.ts

```diff
@@ -47,9 +50,7 @@ export class PinchOnMobileSupport {
     if (this.initialPinchDistance <= 0 || event.touches.length !== 2) {
       return;
     }
-    if (this.pinchScale !== 1) {
-      event.preventDefault();
```

Comment:
Also this was removed as it didn't do anything because we always call `event.preventDefault` at the end of the function and there are no early exits.